### PR TITLE
The Hotswappable Battery by @blaine-t

### PR DESCRIPTION
**Pull request by blaine-t**
## Describe your project in 1-3 sentences

I am planning to build a custom battery bank that has hotswappable batteries so that I can get have a massive capacity and still be able to travel with it. I look forward to learning more about the inner workings of portable battery banks.

## Hack Club Slack handle

@ Blaine Traudt

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: blaine-t.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+name: "@blaine-t"
+project: "The Hotswappable Battery Bank"
+---
+
+# The Hotswappable Battery Bank
+
+## Summary
+
+I am planning to build a custom battery bank that fits my needs and I can use in college and beyond.
```
Comment (bellesea) [inline blaine-t.md:10]: hey @blaine-t :)) you mentioned here that you can "use in college and beyond", does this mean you're currently a high school senior (about to go to college) or are you in college?

**Review by bellesea — CHANGES_REQUESTED** [review]
hii @blaine-t omg this is soo cool wha-

i have no experience w/ hotswappable batteries so out of curousity, what's the role of a PCB in it?

just to check, do you already have access to all the tools/materials you need, like a soldering iron, welder, 3d printer, wires, etc.?

File: blaine-t.md (hunk 1 above)
Comment (blaine-t) [reply blaine-t.md:10]: Yes I am a current high schooler, sorry for the weird wording. I want to use it now and when I'm in college.

**Comment by blaine-t** [comment]
> 

The PCB's role is to make sure that all the cells in the pack maintain the same voltage when charging that way one cell doesn't exceed the safe charging voltage while some cells aren't charged all the way since each cell is slightly different from the factory. The PCB actually does nothing to allow for it to be hotswappable. With the appropriate connector (which I will supply) it should be as easy as plug and play.

And yes I do have all the tools that I listed so I am good to go there!

**Comment by bellesea** [comment]
> > 
> 
> The PCB's role is to make sure that all the cells in the pack maintain the same voltage when charging that way one cell doesn't exceed the safe charging voltage while some cells aren't charged all the way since each cell is slightly different from the factory. The PCB actually does nothing to allow for it to be hotswappable. With the appropriate connector (which I will supply) it should be as easy as plug and play.
> 
> And yes I do have all the tools that I listed so I am good to go there!

ahh coool! thanks sm for expanding on that and for the rapid response hehe :)

**Review by bellesea — APPROVED** [review]
@blaine-t can't wait to see how this turns out!! an awesome member of @hackclub/bank-ops will reach out soon with next steps :)

**Review by MelSmith104 — COMMENTED** [review]
hey @blaine-t this sounds so cool! i can't wait to see it built! im dm'ing you on slack so that we can chat about getting you the grant!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**